src/api/java/io/github/cvc5/Solver.java

```diff
@@ -2315,13 +2401,16 @@ public void blockModel(BlockModelsMode mode)
   private native void blockModel(long pointer, int modeValue);
```

Comment:
The cpp version uses enum `BlockModelsMode`, but the java uses integer. This way users have to remember the values for the enum which is not documented here.